release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -175,6 +175,52 @@ With this release, you can access the NMstate Operator and resources such as the
+This feature changes how the kubelet starts and cleans up orphaned volumes. This change improves how the node and kubelet startup. If you find any issues with this feature, please report them. This feature is supported with Technology Preview status.
+
+[id="ocp-4-14-storage-gcp-filestore-csi-driver"]
+==== GCP Filestore storage CSI Driver Operator is generally available
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Google Compute Platform (GCP) Filestore Storage. The GCP Filestore CSI Driver Operator that manages this driver is now generally available.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-google-cloud-file.adoc[Google Compute Platform Filestore CSI Driver Operator].
+
+[id="ocp-4-14-storage-automigration-vsphere"]
+==== Automatic CSI migration for VMware vSphere
+Starting with {product-title} 4.8, automatic migration for in-tree volume plugins to their equivalent Container Storage Interface (CSI) drivers became available as a Technology Preview feature. 
+
+Support for vSphere was provided in this feature in {product-title} 4.10 at the Technology Preview level, and vSphere support became generally available in 4.13. However, while for new installations of {product-title}, automatic migration was enabled by default, when upgrading from {product-title} 4.12, or earlier, to 4.13, automatic CSI migration for vSphere only occurred if you opted in. In {product-title} 4.14, CSI migration for vSphere is now enabled by default under all circumstances and requires no action by an administrator.
+
+This feature automatically translates in-tree objects to their counterpart CSI representations and should be completely transparent to users.
+
+Although storage class referencing to the in-tree storage plug-in will continue working, it is recommended that you switch the default storage class to the CSI storage class.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-migration.adoc[CSI automatic migration].
+
+[id="ocp-4-14-storage-secrets-store-csi-driver-operator"]
+==== Secrets Store CSI Driver Operator (Technology Preview)
+The Secrets Store Container Storage Interface (CSI) Driver Operator, `secrets-store.csi.k8s.io`, allows {product-title} to mount multiple secrets, keys, and certificates stored in enterprise-grade external secrets stores into pods as a volume. The Secrets Store CSI Driver Operator communicates with the provider using gRPC to fetch the mount contents from the specified external secrets store. After the volume is attached, the data in it is mounted into the container’s file system. This feature is supported with Technology Preview status.
+
+For more information, see ???add link for storage content & authentication content???
+
+[id="ocp-4-14-storage-azure-file-nfs"]
+==== Azure File supporting NFS is generally available
```

Comment:
@lpettyjo 
Azure File CSI NFS support is generally available?

Reply:
@gcharot agreed this is GA, but that there is a known bug that needs to be documented.